usaspending_api/etl/management/commands/copy_table_metadata.py

```diff
@@ -14,7 +14,7 @@
 # since that script uses relative imports which do not work well with the rest of the repo code
 TEMPLATE = {
     "read_indexes": "SELECT indexname, indexdef FROM pg_indexes WHERE schemaname = '{}' AND tablename = '{}';",
-    "read_constraints": "select conname, pg_get_constraintdef(oid) from pg_constraint where contype IN ('p', 'f') and conrelid = '{}'::regclass;",
+    "read_constraints": "select conname, pg_get_constraintdef(oid) from pg_constraint where contype IN ('p', 'f', 'u') and conrelid = '{}'::regclass;",
```

Comment:
I felt like we went through a discussion on this with transaction_search and award_search, and decided to NOT copy unique constraints.

Or ... did we just decide to not copy them for _those_ tables, because they weren't actually necessary on those tables. So we just kept on _not copying them_? 

If the latter, what is it about the unique constraints you're seeing that makes you feel like we ought to keep them, rather than discard them like we did for award_search and transaction_search?